### ISSUE TITLE
LoadingSpinner status is not announced by screen readers until loading is done.

### ISSUE THREAD

**Issue by danielck** [vrk-kpa/suomifi-ui-components#737]
## Expected Behavior

When doing a simulated load of the LoadingSpinner ("Simuloi lataus") on the Design System dev website (https://vrk-kpa.github.io/suomifi-design-system/components/loadingspinner/), screen readers should announce the text "Loading".

## Actual Behavior

When tested with NVDA and Firefox on Windows, only the success/error messages are announced ("Lataus onnistui", "Lataus epäonnistui".) This is probably due to the use of aria-busy, which is designed to block any updates to the region while the value is `true`.

## Steps to Reproduce the Problem

1. Have NVDA running, using Firefox. Issue probably exists in other browsers too. Load https://vrk-kpa.github.io/suomifi-design-system/components/loadingspinner/

2. Click "Simuloi lataus".

## Possible fix

The aria-live area should be outside the region marked with aria-busy.

**Comment by LJKaski** [comment]
This is on purpose and it was discussed with multiple accessibility experts at the time of creation. The rationale being that most of the time the loaded content appears elsewhere on the page than where the current focus is, and the load times are short enough that it does not make sense to announce the loading state. There might even be multiple loading indicators present at the same time and it would be a mess if all of them announce the loading state via aria-live. With this implementation the idea is that if a user navigates to a container the content of which is still being fetched/loaded, they get the information at that time, but not otherwise.
